eslint.config.mjs

```diff
@@ -568,4 +569,36 @@ export default tseslint.config(
       'import/no-default-export': 'off',
     },
   },
+  {
+    ...perfectionistPlugin.configs['recommended-alphabetical'],
```

Comment:
```suggestion
    extends: [perfectionistPlugin.configs['recommended-alphabetical']],
```

Reply:
I didn't know this was even supported any more, it is not listed in the configuration docs on the eslint site

Reply:
@JamesHenry It's not part of ESLint - it's part of our config util 😄 
https://typescript-eslint.io/packages/typescript-eslint#flat-config-extends

I added it to our util because doing this sort of thing is so common that manually doing what Josh had to do (manually pull apart an object... or is it an array... who knows!)